rust/osm2lanes/src/transform.rs

```diff
@@ -70,8 +110,7 @@ fn non_motorized(tags: &Tags, cfg: &Config) -> Option<LaneSpecResult> {
     // types, assume bikes are allowed, except for footways, where they must be explicitly
     // allowed.
     if tags.is("bicycle", "no")
-        || (tags.highway_is("footway")
-            && !tags.is_any("bicycle", &["designated", "yes", "dismount"]))
+        || (tags.highway_is("footway") && !tags.is_any("bicycle", &["designated", "yes"]))
```

Comment:
What happened to dismount?

Reply:
I wasn't convinced it was correct...

```
highway=footway
bicycle=dismount
```

That's a sidewalk, or am I missing something?

Reply:
Ahh nope, you're right. Originally I was hacking around stuff in A/B Street by treating this as a cycle lane, to make sure stuff was connected. That hack makes no sense here.

https://taginfo.openstreetmap.org/tags/bicycle=dismount#combinations
For reference, there are some ways that use both these tags -- so they kind of indicate bike use is explicitly allowed, but only with dismounting. Not sure how to fit that in this spec yet.

Reply:
I think it's a bit more useful for routers, to give the way a low speed but high desirability for cyclists?

Reply:
Yeah, makes sense. I've experienced a few of these around Vauxhall in London particularly, where the intended design is probably to stop and act like a pedestrian briefly.

I wonder how to reflect this in the spec. Maybe instead of access per vehicle type being binary, some kind of "allowed, but with a penalty"?

Reply:
depends what the libraries clients are interested in.
I wonder if most just don't care, and would be more happy pretending it is a pedestrian way.
In the draft of the spec I have, I use "designated" to give a hint on how to draw the lane.

I would later want to extend the spec with more yes/no/designated/dismount information per lane.
I guess that is what you need for abstreet? essentially lane by lane routing?
so then have access tags returned for a lane mimic the access tags for a one lane highway?

Reply:
> I wonder if most just don't care, and would be more happy pretending it is a pedestrian way.

On second thought, this makes more sense to me. No need to complicate things before a particular client needs it.